addons/website_slides/static/src/xml/slide_quiz.xml

```diff
@@ -7,9 +7,14 @@
                 <div t-foreach="widget.quiz.questions" t-as="question"
                      t-attf-class="o_wslides_js_lesson_quiz_question mt-5 mb-4 #{widget.slide.completed ? 'completed-disabled' : ''}"
                      t-att-data-question-id="question.id" t-att-data-title="question.question">
-                    <div class="h4">
-                        <small class="text-muted"><span t-esc="question_index+1"/>. </small> <span t-esc="question.question"/>
-                    </div>
+                     <div class="d-flex align-items-baseline">
+                        <div class="h4">
```

Comment:
Would using "inline-block" on `<h4>` and `<small>` work to simplify this and avoid a container?